include/LoopClosing.h

```diff
@@ -47,8 +47,12 @@ class LoopClosing
 public:
 
     typedef pair<set<KeyFrame*>,int> ConsistentGroup;    
+    // typedef map<KeyFrame*,g2o::Sim3,std::less<KeyFrame*>,
+    //     Eigen::aligned_allocator<std::pair<const KeyFrame*, g2o::Sim3> > > KeyFrameAndPose;
+    
+    //Mac support for ORB-SLAM3
     typedef map<KeyFrame*,g2o::Sim3,std::less<KeyFrame*>,
-        Eigen::aligned_allocator<std::pair<const KeyFrame*, g2o::Sim3> > > KeyFrameAndPose;
+            Eigen::aligned_allocator<std::pair<KeyFrame *const, g2o::Sim3> > > KeyFrameAndPose;
```

Comment:
@match08 const KeyFrame* is not the same as KeyFrame *const and we need the first one here!

Reply:
The change looks correct to me. I encountered the same issue building on Windows, and this change was the fix for me as, well.